<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File Information</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="lblWorkingDir">
       <property name="text">
        <string>Working dir:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="pbWorkingDir">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normalon>../../../../usr/local/share/icons/gnome/32x32/places/folder.png</normalon>
        </iconset>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="pbCommand">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normalon>../../../../usr/local/share/icons/gnome/32x32/actions/gnome-run.png</normalon>
        </iconset>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QCheckBox" name="cbStartupNotification">
       <property name="text">
        <string>Use startup notification</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lWorkingDir">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="lblIcon">
       <property name="text">
        <string>Icon:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="lblCommand">
       <property name="text">
        <string>Command:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="lblComment">
       <property name="text">
        <string>Comment:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lCommand"/>
     </item>
     <item row="4" column="1" alignment="Qt::AlignLeft">
      <widget class="QPushButton" name="pbIcon">
       <property name="maximumSize">
        <size>
         <width>275</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normalon>../../../../usr/local/share/icons/gnome/32x32/categories/xfce-graphics.png</normalon>
        </iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QCheckBox" name="cbRunInTerminal">
       <property name="text">
        <string>Run in terminal</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="lblName">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="lblOptions">
       <property name="text">
        <string>Options</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLineEdit" name="lName"/>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLineEdit" name="lComment"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbApply">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lName</tabstop>
  <tabstop>lComment</tabstop>
  <tabstop>lCommand</tabstop>
  <tabstop>lWorkingDir</tabstop>
  <tabstop>pbIcon</tabstop>
  <tabstop>cbStartupNotification</tabstop>
  <tabstop>cbRunInTerminal</tabstop>
  <tabstop>pbCancel</tabstop>
  <tabstop>pbApply</tabstop>
  <tabstop>pbCommand</tabstop>
  <tabstop>pbWorkingDir</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pbCancel</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>289</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbApply</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>375</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
